src/api/status/public/index.html

```diff
@@ -1307,5 +1328,6 @@ <h6 class="mt-3">Thank you for sharing!</h6>
     <!-- Telescope JS Files -->
     <script src="js/github-stats.js"></script>
     <script src="js/feed/index.js"></script>
+    <script type="module" src="js/index.js"></script>
```

Comment:
Can we please follow the same pattern as these other two files for naming and type?

Reply:
@nguyenhung15913 this should be changed to js/pages/dashboard.js